apps/indexer/config/config.exs

```diff
@@ -91,7 +94,14 @@ config :logger, :indexer, backends: [LoggerJSON]
 #  metadata_filter: [application: :indexer]
 
 config :indexer, Indexer.Block.Fetcher, enable_gold_token: true
+config :indexer, Indexer.Prometheus.MetricsCron, metrics_fetcher_blocks_count: 1000
+config :indexer, Indexer.Prometheus.MetricsCron, metrics_cron_interval: 2
 
 # Import environment specific config. This must remain at the bottom
 # of this file so it overrides the configuration defined above.
 import_config "#{Mix.env()}.exs"
+
+config :prometheus, Indexer.Prometheus.Exporter,
+  path: "/metrics/indexer",
```

Comment:
For some reason, I see nothing at "/metrics/indexer":

<img width="1084" alt="Screenshot 2021-06-25 at 14 47 31" src="https://user-images.githubusercontent.com/4341812/123420533-53baf600-d5c4-11eb-8869-c90d8344cb7a.png">


Though, I see indexer metrics in `/metrics` path

Reply:
Hey Victor, they should be available under: localhost:4001/metrics/indexer (or the port you specified when starting the indexer). I will change the web metrics to be available under /metrics/web for consistency.